src/services/api-service.ts

```diff
@@ -72,36 +92,23 @@ class ApiService {
   }
 
   async sendQueryRequest(query: string, props: any, structuredLogInfo?: StructuredLogInfo): Promise<any> {
-    if (this.logger.structuredLog !== undefined && structuredLogInfo !== undefined) {
-      this.logger.structuredLog(
-        structuredLogInfo.network,
-        `Sending query request ${query} with props ${JSON.stringify(props)}`,
-        structuredLogInfo.tagId,
-      )
-    } else {
-      this.logger.log(`Sending query request ${query} with props ${JSON.stringify(props)}`)
-    }
-
+    this.logInfo(`Sending query request ${cleanRequest(query)} with props ${JSON.stringify(props)}`, structuredLogInfo)
     try {
-      return await this.client.request(query, props)
+      return await this.client.rawRequest(query, props)
```

Comment:
what does `rawRequest` do differently?

Reply:
It returns a more complete response, with `{ data, errors, extensions, headers, status }` instead of just the data.

Reply:
@BravoNatalie Can you set the type return for `sendQueryRequest` so we are not using any.